jdaviz/configs/default/plugins/subset_tools/tests/test_subset_tools.py

```diff
@@ -276,8 +277,7 @@ def test_get_regions(cubeviz_helper, spectrum1d_cube, imviz_helper):
         plg.get_regions(region_type='fail')
 
 
-@pytest.mark.xfail(reason='Unskip once issue XXXX is resolved.')
-def test_get_regions_composite(cubeviz_helper, spectrum1d_cube):
+def test_get_regions_composite(imviz_helper):
```

Comment:
This test does not need Cubeviz. Imviz should be enough. Also `spectrum1d_cube` dimension is too small to actually meaningfully contain the regions we are creating in the test, so it should not be used for this test.